### ISSUE TITLE
Test execution freezes on Windows 10 nodeJS 10

### ISSUE THREAD

**Issue by ngeor** [yeoman/yeoman-test#51]
Possibly related issues: #43 and #15 
Environment: Windows 10, nodeJS 10.14.1 (LTS), mocha

When I have just a single unit test, where the `helpers.run` occurs only once, everything is fine.

Example:

```
    before(() => helpers.run(__dirname)
      .withPrompts({
        name: 'mylib',
        description: 'my lib is awesome',
      }));
```

When I add a new `describe` with a different set of prompts, the tests never complete, it just stays there frozen.

Example:

```
  describe('unscoped', () => {
    before(() => helpers.run(__dirname)
      .withPrompts({
        name: 'mylib',
        description: 'my lib is awesome',
      }));

    it('should create the expected files', () => {
      assert.file([
        '.editorconfig',
        '.eslintrc.js',
      ]);
    });

    it('should have the expected package name', () => {
      assert.jsonFileContent('package.json', {
        name: 'mylib',
      });
    });
  });

  describe('scoped', () => {
    before(() => helpers.run(__dirname)
      .withPrompts({
        name: 'mylib2',
        description: 'my lib is awesome',
        scope: '@ngeor',
      }));
```

I can experience the same problem by changing the `before` to `beforeEach` in the first `describe`.

I spent a lot of time trying to troubleshoot it with no success. This happens also with jest (I tried it with the generator that is generated by generator-generator).

Then I found issues #43 and #15 and I tried to run my tests within Docker. Everything worked fine... also my build in Travis runs fine, so this is a Windows-specific issue.

My temporary workaround is to run the tests locally with Docker, but this slows things down.

My generator is this one https://github.com/ngeor/generator-nodejs

**Comment by gavinr** [comment]
I am seeing this issue too. 

- Windows 10 Enterprise
- Node: v10.15.3
- NPM: v6.4.1

**Comment by mshima** [comment]
Duplicate of https://github.com/yeoman/yeoman-test/issues/43.
Fixed on inquirer.
